app/controllers/passengers_controller.rb

```diff
@@ -0,0 +1,74 @@
+class PassengersController < ApplicationController
+
+  def index
+    @passengers = Passenger.all.order(:id)
+  end
+
+  def show
+    passenger_id = params[:id].to_i
+    @passengers = Passenger.find_by(id: passenger_id)
+
+    if @passengers.nil?
+      head :not_found
+      return
+    end
+  end
+
+  def new
+    @passengers = Passenger.new
+  end
+
+  def create
+    @passengers = Passenger.new(passengers_params)
+    if @passengers.save # save returns true if the database insert succeeds
+      redirect_to passengers_path(@passengers.id) # go to the index so we can see the task in the list
```

Comment:
This is why your test is failing

```suggestion
      redirect_to passenger_path(@passengers.id) # go to the index so we can see the task in the list
```